### DISCUSSION TITLE
The search bar doesn't handle IME composition events properly

### DISCUSSION THREAD

**Discussion by RexSkz** [community/community#37416] [Other Feature Feedback, Questions, & Ideas]
I was typing the word `encodeURIComponent` but I mistakingly used a Chinese IME...

<img width="352" alt="image" src="https://user-images.githubusercontent.com/27483702/198176583-815a1bcf-2cf1-4060-a1c6-9c5cc2d7516d.png">

To make sure what I typed was English, I pressed Enter key and... triggered the search action, with the broken words.

<img width="768" alt="image" src="https://user-images.githubusercontent.com/27483702/198176746-9c8799d5-4dad-4c75-b771-a8c2244b6510.png">

This issue is not critical, but it affects the user experience for the users who accidentally use IME. Maybe it can be fixed in the future :)

**Comment by ossanna16** [comment]
Hi there @RexSkz and welcome to our community! Thank you for reporting this 🙂

**Reply by mooleetzi** [reply]
same question with typing English words in Chinese Input
